lalframe open_data_source test fails if gwpy installed in a symbolic link

Some of the `gwpy/timeseries/tests/test_io_gwf_lalframe.py` tests fail when GWpy is installed into a symbolic link:

```text
timeseries/tests/test_io_gwf_lalframe.py FFFFF...........                                       [ 68%]
```

The failures are of the form

```text
    def _test_open_data_source(source):
        """This function actually performs the test
        """
        stream = gwpy_lalframe.open_data_source(source)
        assert stream.epoch == TEST_GWF_SEGMENT[0]
>       assert TEST_GWF_PATH == Path(
            urlparse(stream.cache.list.url).path
        ).absolute()
E       AssertionError: assert PosixPath('/cvmfs/oasis.opensciencegrid.org/ligo/sw/conda/envs/igwn-testing/lib/python3.9/site-packages/gwpy/testing/data/HLV-HW100916-968654552-1.gwf') == PosixPath('/cvmfs/oasis.opensciencegrid.org/ligo/deploy/sw/conda/envs/igwn-py39-testing/lib/python3.9/site-packages/gwpy/testing/data/HLV-HW100916-968654552-1.gwf')
```

Somewhere I think we need to unwrap the symlinks on both sides of the assertion.

--- a/gwpy/timeseries/tests/test_io_gwf_lalframe.py
+++ b/gwpy/timeseries/tests/test_io_gwf_lalframe.py
@@ -61,9 +61,7 @@ def _test_open_data_source(source):
     """
     stream = gwpy_lalframe.open_data_source(source)
     assert stream.epoch == TEST_GWF_SEGMENT[0]
-    assert TEST_GWF_PATH == Path(
-        urlparse(stream.cache.list.url).path
-    ).absolute()
+    assert Path(urlparse(stream.cache.list.url).path).samefile(TEST_GWF_PATH)
 
 
 @pytest.mark.parametrize("source", [

